### PR TITLE
I fixed the formatting, got rid of unnecessary code and edited the sel…

### PR DESCRIPTION
**Pull request by Halezmarie**
…ector so that the toy would display.

### CODE REVIEW

**Comment by Halezmarie** [comment]
I changed `usehistory` to` redirect` because it handles the toy name not being defined when being deleted and because it also handles sending it back to the toys correctly without any errors. 

I added `if (toy)`  with the `redirect` because by the time the user hits delete, the toy ends up being undefined. So it redirects back to toys instead of having the user stare at a blank page.

File: src/components/DisplayToy.js [hunk 1]
```diff
@@ -4,29 +4,32 @@ import { Container, Button } from 'react-bootstrap';
...
+          <Button onClick={() => dispatch(deleteToy(toy.id))}>
+            Delete Toy
+          </Button>
+          <ReviewsContainer toy={toy} />
+        </div>
+      </Container>
+    );
+  else {
+    return <Redirect to="/toys" />;
+  }
```
Comment (ssteele-godaddy) [inline src/components/DisplayToy.js:32]: This is personal preference, but if I have a long `if` block and else is just a little one-liner, I find it easier to read if you short circuit earlier.  For example:

```
if (!toy) return <Redirect to="/toys" />;

return(
  <Container>
    ...
  </Container>
);
```

File: src/selectors/toySelectors.js [hunk 2]
```diff
@@ -1,8 +1,5 @@
 export const selectToys = ({ toys }) => toys;
-export const selectToy =
-  (toyId) =>
-  ({ toys }) =>
-    toys.find((toy) => toy.id === toyId);
+export const selectToy = (toyId) => ({ toys }) => toys.find((toy) => toy.id === Number(toyId));
```
Comment (ssteele-godaddy) [inline src/selectors/toySelectors.js:2]: 👍

File: src/components/DisplayToy.js (hunk 1 above)
Comment (ssteele-godaddy) [inline src/components/DisplayToy.js:32]: If you're leaving as is, this looks a little weird that there's only one set of {}, I'd expect to see 2 sets of {}, one for the `if`, the other for the `else`.

What's there now:
```
if (toy)
  ...
else {
  ...
}
```

...I'd propose to make things more consistent:

```
if (toy) {
 ...
} else {
  ...
};
```

**Review by ssteele-godaddy — CHANGES_REQUESTED** [review]
Looks good, just one thing here that could use a little bit of attention.

File: src/components/DisplayToy.js (hunk 1 above)
Comment (Halezmarie) [reply src/components/DisplayToy.js:32]: It definitely looks better like that!

Comment (Halezmarie) [reply src/components/DisplayToy.js:32]: I am going to make the change you suggested because a one liner looks a lot cleaner for sure. Thanks!

**Pull request merged**